src/cogs/interactions/slash/mongo.py

```diff
@@ -0,0 +1,72 @@
+import discord
+from discord import app_commands
+from discord.ext import commands
+from pymongo import MongoClient
+import json
+import os
+
+
+with open("config.json") as f:
+    config = json.load(f)
+
+mongo_client = MongoClient(os.getenv("MONGO_URI"))
+db = mongo_client.get_database()
+verified_collection = db["verified"]
+batch_collection = db["batch"]
+
+
+class Verification(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @app_commands.command(name="info", description="Get verification info about a user.")

```

Comment:
let's rewrite this one. allow any verified person to run /info. return an embed with basic discord info about the user (username, profile pic, when they verified, and anything else you can think about). now if the person who ran the /info command is a mod/admin, send a follow-up ephermal message/embed with the privileged info, which is only PRN. lemme know if you need more clarification on this